ledfx/virtuals.py

```
@@ -779,6 +831,113 @@ def force_frame(self, color):
+
+        Returns:
+            (spatial_frame, gradient_lut) where:
+            - spatial_frame: N×3 float array, gradient sampled at pixel positions.
+              Used for solid colors, and as the static "preview" for gradients.
+            - gradient_lut: 1024×3 float array sampled uniformly 0→1, or None for
+              solid colors. Used at render time: per-pixel hue → LUT index → color.
+        """
+        if self._color_override is None or self.pixel_count == 0:
+            return None, None
+
+        from ledfx.color import RGB, Gradient, parse_gradient
```

Flake8 (7.3.0)

[error] 'ledfx.color.Gradient' imported but unused

(F401)